src/lib/components/InputGroup/__tests__/InputGroup.test.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+  render,
+  within,
+} from '@testing-library/react';
+import { Button } from '../../Button';
+import { SelectField } from '../../SelectField';
+import { TextField } from '../../TextField';
+import { childrenEmptyPropTest } from '../../../../../tests/propTests/childrenEmptyPropTest';
+import { isLabelVisible } from '../../../../../tests/propTests/isLabelVisible';
+import { layoutPropTest } from '../../../../../tests/propTests/layoutPropTest';
+import { InputGroup } from '../InputGroup';
+
+const mandatoryProps = {
+  children: [
+    <TextField key="text field" label="text field" />,
+    <SelectField
+      key="select field"
+      label="select field"
+      options={[{
+        label: 'option 1',
+        value: 1,
+      }]}
+    />,
+    <Button key="button" label="button" />,
+  ],
+  label: 'label',
+};
+
+describe('rendering', () => {
+  it.each([
+    ...childrenEmptyPropTest,
+    [
```

Comment:
Do we need this test case. I think it is covered by the test case bellow. If we confirm, that the element has a class, we also know it exists.